docs/Reference/JSON-RPC-API-Methods.md

```diff
@@ -0,0 +1,1911 @@
+description: Pantheon JSON-RPC API methods reference
+<!--- END of page meta data -->
+
+# JSON-RPC API Methods
+
+!!! danger "Breaking Change in v0.8.3"
+    From v0.8.3, incoming HTTP requests are only accepted from hostnames specified using the [`--host-whitelist`](Using-JSON-RPC-API.md#breaking-change-in-v083) option. 
+
+The following lists the Pantheon JSON-RPC API commands:
+
+## Admin Methods
+
+### admin_peers
+
+Returns networking information about connected remote nodes. 
+
+**Parameters**
+
+None
+
+**Returns**
+
+`result` : *array* of *objects* - Object returned for each remote node. 
+
+Properties of the remote node object are:
+
+* `version` - P2P protocol version
+* `name` - Client name
+* `caps` - P2P message capabilities 
+* `network` - Addresses of local node and remote node
+* `port` - Port
+* `id` - Node public key. Excluding the `0x` prefix, the node public key is the ID in the enode URL `enode://<id ex 0x>@<host:port>`. 
+
+!!! example
+    ```bash tab="curl HTTP request"
+    $ curl -X POST --data '{"jsonrpc":"2.0","method":"admin_peers","params":[],"id":1}' <JSON-RPC-http-endpoint:port>
+    ```
+    
+    ```bash tab="wscat WS request"
+    $ wscat -c <JSON-RPC-ws-endpoint:port>
+    connected (press CTRL+C to quit)
+    > {"jsonrpc":"2.0","method":"admin_peers","params":[],"id":1}
+    ```
+    
+    ```json tab="JSON result"
+    {
+      "jsonrpc" : "2.0",
+      "id" : 1,
+      "result" : [ 
+        {
+          "version" : "0x5",
+          "name" : "Geth/v1.8.20-unstable-92639b67/linux-amd64/go1.11.1",
+          "caps" : [
+            "eth/62", 
+            "eth/63"
+            ],
+          "network" : {
+            "localAddress" : "192.168.1.229:51279",
+            "remoteAddress" : "52.3.158.184:30303"
+          },
+          "port" : "0x0",
+          "id" : "0x343149e4feefa15d882d9fe4ac7d88f885bd05ebb735e547f12e12080a9fa07c8014ca6fd7f373123488102fe5e34111f8509cf0b7de3f5b44339c9f25e87cb8"
+        } 
+      ]
+    }
+    ```
+
+## Web3 Methods
+
+### web3_clientVersion
+
+Returns the current client version.
+
+*Parameters*
+
+None
+
+*Returns*
```

Comment:
This is bold for admin_peers - I think it's better bold when scanning the ref. This comment applies to all of the methods (ie, I think Returns should be changed to bold throughout)

Reply:
same